smbackend/exception_handler.py

```diff
@@ -0,0 +1,28 @@
+import json
+
+from django.db.utils import InternalError
+from django.http.response import HttpResponseBadRequest
+from rest_framework.views import exception_handler
+from rest_framework.exceptions import ValidationError
+
+
+def custom_exception_handler(exc, context):
+
+    # Call REST framework's default exception handler first,
+    # to get the standard error response.
+    response = exception_handler(exc, context)
+    json_resp = json.dumps(response.data) if response else exc
+
+    if isinstance(exc, ValidationError):
+        return HttpResponseBadRequest(content=json.dumps('ValidationError: {}'.format(json_resp)),
```

Comment:
This is not what we want. It technically does return valid JSON, but it's a single string! You're converting a python string to a json string. What we want is to convert a python dict to a json object.